### PR TITLE
🚑️ 617 - make pub urls clickable in pdf

### PR DESCRIPTION
**Pull request by anncatton**
Hotfix - daco folks requested pub urls be clickable in the pdf, and we wanted to avoid the attestation changes that have been merged into dev since the last release.

### CODE REVIEW

File: components/pages/Applications/PDF/StaticProjectInfo.tsx [hunk 1]
```diff
@@ -218,13 +201,17 @@ export const LaySummaryBubble = ({ isPdf = false }: { isPdf?: boolean }) => {
 const PdfPublicationsFormData = ({ data = [] }: { data?: string[] }) => {
   const pubData = data.map((d: string) => ({
     fieldName: PdfFieldName.PUBLICATION_URL,
-    fieldValue: d,
+    fieldValue: (
+      <Link src={d} style={{ textDecoration: 'underline' }}>
+        {d}
+      </Link>
+    ),
```
Comment (joneubank) [inline components/pages/Applications/PDF/StaticProjectInfo.tsx:208]: 👍

**Review by joneubank — APPROVED** [review]
👩‍⚕️ 💉🤕

**Review by caravinci — APPROVED** [review]
I may be mistaken, but this looks like only a couple meaningful simple changes (2 o 3 lines), with a few `prettier` things? (please correct me if I missed anything)

J'approve! 👍

**Comment by anncatton** [comment]
> I may be mistaken, but this looks like only a couple meaningful simple changes (2 o 3 lines), with a few `prettier` things? (please correct me if I missed anything)
> 
> J'approve! 👍

Yep it's just those pub changes and the rest is formatting! thanks for checking

**Pull request merged**